docs/06_how-to-guides/40_multi-index/how-to-delete-data-from-a-multi-index-table.md

```diff
@@ -1,27 +1,51 @@
 ---
-content_title: How to delete data from a multi index table
+content_title: How to delete data from a multi-index table
 ---
 
-## Preconditions
-- It is assumed you already have a multi index table instance defined along with its mandatory primary index, otherwise take a look at the section [How to instantiate a multi index table](./how-to-instantiate-a-multi-index-table.md).
+## Overview
 
-To delete data from a multi index table follow the steps below:
+This guide provides instructions to to delete data from a multi-index table.
+
+## Reference
+
+See the following code reference:
+
+* The [`multi-index`](../../classeosio_1_1multi__index) class.
+* The [`multi-index::find(...)`](../../group__multiindex#function-find) method.
+* The [`multi-index::erase(...)`](../../group__multiindex/#function-erase) method.
+
+## Before you begin
+
+Make sure you have the following prerequisites in place:
+
+* An EOSIO development environment, for details consult the [Get Started Guide](https://developers.eos.io/welcome/latest/getting-started-guide/index),
+* A multi-index `testab` table instance which stores `user` objects indexed by the primary key which is of type `eosio::name`. Consult the section [How to instantiate a multi-index table](./how-to-instantiate-a-multi-index-table.md) to learn how to set it up.
+
+## Procedure
+
+Complete the following steps to implement a `del` action which deletes an user object, identified by its account name, from the multi-index table.
+
+### 1. Find The User You Want To Delete
+
+Make use of the multi-index [`find(...)`](../../group__multiindex#function-find) method to locate the user object you want to delete. The targeted user is searched based on its account name.
```

Comment:
Make use of the multi-index [`find(...)`](../../group__multiindex#function-find) method to locate the user object you want to delete. The targeted user is searched based on its account name.

=>

Use the multi-index [`find(...)`](../../group__multiindex#function-find) method to locate the user object you want to delete. The targeted user is searched based on its account name.

Reply:
I got rid of "make use of" from everywhere